requirements/rpms.txt

```diff
@@ -12,8 +12,8 @@ openldap-devel-2.4.46
 # Dependency for SSL support in the python-qpid-proton Python package
 openssl-1.1.1k
 openssl-devel-1.1.1k
-python39-3.9.13-1.module+el8.7.0+15656+ffd4a257
```

Comment:
Martin and I discussed this yesterday when some old RPMs where removed from the UBI-8 RPM repos that we should probably not set the version as at, and just use for example `python-39` here. Otherwise we might miss some security updates, and also the builds will break again when these RPMs are purged. It shouldn't have any compatibility consequences.

Reply:
Ack, I'll tweak this. Originally Judy Kelly recommended we pin the versions we use: https://docs.google.com/document/d/1n-4CU_Mim8Fauy0iFzelLLy4QaGy747VF5cMU60txDU/edit#heading=h.a2f0mozgq7rz

It's not worth it though since those package versions can get removed. I thought they'd be stable for the lifetime of a particular UBI8 version, and only backported security updates (with the same version and a newer release) would be added.